deviations/tests.py

```diff
@@ -12,6 +12,10 @@
 
 class DeviationsTest(TestCase):
     def setUp(self):
+        self.teacher = User(username="staff", is_staff=True)
```

Comment:
Doesn't really matter, but I doubt `is_staff=True` is needed here. It allows access to the Django admin site. It is not related to A+ or "course staff".

https://docs.djangoproject.com/en/3.2/ref/contrib/auth/#django.contrib.auth.models.User.is_staff

Reply:
Ok, I just copied the rows related to `self.teacher` from `exercise/tests.py`. I suppose the `is_staff` parameter is not needed there either since that file doesn't test the API.